broker-daemon/broker-rpc/orderbook-service/get-orderbook.spec.js

```diff
@@ -77,4 +85,41 @@ describe('getOrderbook', () => {
         timestamp: '1537487471431784972' }
     )
   })
+
+  it('limits the number of orders', async () => {
```

Comment:
should add more `expect`s about calls to `getOrders` and the parameters you are expecting